torch/csrc/jit/codegen/cuda/executor.cpp

```diff
@@ -303,7 +344,7 @@ uint64_t FusionExecutor::computeSharedMemory(
     const std::vector<const kir::Allocate*>& buffers,
     bool align_padding,
     uint64_t total) {
-  FUSER_PERF_SCOPE("computeSharedMemory");
+  FUSER_PERF_SCOPE("computeSharedMemory::old");
```

Comment:
Do we need to keep this old version?

Reply:
Will try to do that in follow up change, if performance regression wouldn't be too bad. Other thing would be getting rid of all uses of old evaluator (building expression in place and evaluating them in place).

Reply:
This is still work in progress, not included in current version